### DISCUSSION TITLE
Trouble getting buttons the same size

### DISCUSSION THREAD

**Discussion by stevepurcell** [tailwindlabs/tailwindcss#2263] [Help]
I am relatively new to tailwindcss but making good progress and using it exclusively.

I have a table with two buttons, one is a true submit button for a delete, and the other is an href tag.  I am trying to get the buttons the same size but haven't had any luck.

They are close, but not exact.

![Screen Shot 2020-08-27 at 4 30 57 PM](https://user-images.githubusercontent.com/48142741/91497010-cc590500-e882-11ea-9119-2c2e140440fb.png)

Here is my css.  I dropped the padding down to py-1 and px-3 on the delete button to get it as close as it is.

```
<td  class="p-2" width="5px">
    <a href="drivers/{{ $driver->id }}/edit" 
                  class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded">Edit</a>
</td>
<td  class="p-2" width="5px">
    <form method="post" action="{{ route('drivers.delete', $driver->id) }}">
                {{ method_field('delete') }}
               <button type="submit" 
                   class="bg-red-500 hover:bg-red-700 text-white font-bold py-1 px-3 rounded">Delete</button>
               {{ csrf_field() }}
    </form>
</td>

**Comment by vms82** [comment]
@stevepurcell add whatever display property fit your needs to the anchor tag and use width utils that satisfy you, same width on button & anchor tag, no need to shrink padding to adjust for this
for example "block w-18"
